gradle.properties

```diff
@@ -10,7 +10,7 @@ minecraft_version=1.20.2
 # Check for latest at https://maven.architectury.dev/architectury-plugin/architectury-plugin.gradle.plugin
 architectury_plugin_version=3.4-SNAPSHOT
 # Check for latest at https://maven.architectury.dev/dev/architectury/loom/dev.architectury.loom.gradle.plugin
-architectury_loom_version=1.4-SNAPSHOT
+architectury_loom_version=1.5-SNAPSHOT
```

Comment:
I know fabric-loom recently released 1.6, but maybe architectury is lagging behind a bit?

Reply:
Arch loom 1.6 is beta only.